pvtrace/Trace.py

```diff
@@ -32,6 +32,43 @@
 import pdb
 from copy import copy
 import logging
+import time
+
+# ---- For parallelising: ---- #
+""" You must download the Parallel Python module:
+http://www.parallelpython.com/
+"""
+
+sys.path.append("C:\python27")
+sys.path.append("C:\python27\pp-1.6.4")
```

Comment:
Again, Window's specific hard coded paths, should not be committed, they should be configured outside the code base.